Microsoft.Toolkit.Uwp.UI.Controls/Microsoft.Toolkit.Uwp.UI.Controls.csproj

```diff
@@ -20,6 +20,19 @@
     <None Include="VisualStudioToolsManifest.xml" Pack="true" PackagePath="tools" />
     <None Include="$(OutDir)\Design\$(MSBuildProjectName).Design.dll" Pack="true" PackagePath="lib\$(TargetFramework)\Design" />
   </ItemGroup>
+ 
+  <ItemGroup>
```

Comment:
Don't think we need this extra ItemGroup?

Reply:
Are you referring to the ImageCropper Item Group? This belongs to the ImageCropper Control.

Reply:
Yeah, I just don't know why it appeared in this PR as this isn't for image cropper.  I feel like they're from someone opening these up in the designer, as they should already be included as part of the project style we're using.